### ISSUE TITLE
[FEATURE] Gitlab support?

### ISSUE THREAD

**Issue by kir0ul** [conda-incubator/grayskull#345]
**Is your feature request related to a problem? Please describe.**
If I'm not mistaken, recipe generation from Gitlab is not supported, would it be possible to support it?

**Describe the solution you'd like**
I would like the following exxample command to work as it works with Github:
```
greyskull pypi https://gitlab.com/beenje/jupyterlab-gitlab
```

**Comment by marcelotrevisani** [comment]
indeed, that is not supported and it should be relatively easy to add
